provision/terraform/uat/casper/terraform.tfvars

```diff
@@ -0,0 +1,21 @@
+name = "casper"
+
+bill-to = "Casper"
+
+app-instance-ssh-user = "ubuntu"
+app-instance-ssh-key-name = "CasperEnvDockerHost"
+
+docker-ansible-dir = "../../../ansible/"
+
+docker-tls-port = "2376"
+
+app-dns-name = "casper.a.openlmis.org"
+app-use-route53-domain = true
+app-route53-zone-name = "a.openlmis.org"
+aws-tls-cert-domain = "*.a.openlmis.org"
+
+app-instance-group = "docker-hosts"
+
+olmis-db-instance-class = "db.t2.medium"
```

Comment:
This and the two entries below it should probably be cleared (similar to demo-v3) if we are using the db container.